source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -142,15 +142,31 @@ def wrapper(*args, **kwargs):
                     allowed_ids.update(args.get('allowed_id[]', []))
                     args['allowed_id[]'] = list(allowed_ids)
                     args['_type'] = 'journeys'
-                    args['rel'] = 'same_journey_schedules'
 
                     # Delete arguments that are contradictory to the 'same_journey_schedules' concept
                     if '_final_line_filter' in args:
                         del args['_final_line_filter']
                     if '_no_shared_section' in args:
                         del args['_no_shared_section']
 
-                    journey['links'] = [create_external_link('v1.journeys', **args)]
+                    # Add datetime depending on datetime_represents parameter
+                    if 'datetime_represents' not in args:
+                        args['datetime'] = journey['departure_date_time']
+                    else:
+                        args['datetime'] = (
+                            journey['departure_date_time']
+                            if 'departure' in args.get('datetime_represents')
+                            else journey['arrival_date_time']
+                        )
```

Comment:
```suggestion
                    datetime_repr = args.get('datetime_represents')
                    args['datetime'] = {
                        'departure' : journey['departure_date_time'],
                        'arrival' : journey['arrival_date_time'],
                    }.get(datetime_repr, journey['departure_date_time'])
```


It was very hard to read the nested conditions in my opinion.
Maybe something along those lines could help readability ?

Reply:
:+1: